config/crd/bases/tower.ansible.com_ansibleexecutionenvironments.yaml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: ansibleexecutionenvironment.tower.ansible.com
+spec:
+  group: tower.ansible.com
+  names:
+    kind: AnsibleExecutionEnvironment
+    listKind: AnsibleExecutionEnvironmentList
+    plural: AnsibleExecutionEnvironments
+    singular: AnsibleExecutionEnvironment
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: (Tech Preview) AnsibleExecutionEnvironment is the Schema for the ansibleexecutionenvironment API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              name:
+                type: string
+                description: The name for the executionenvironment
+              new_name:
+                type: string
+                description: The new name for the executionenvironment in case of a name change
+              credential:
+                type: string
+                description: The container registry credential for pulling the executionenvironment image
+              description:
+                type: string
+                description: The description for the executionenvironment
+              image:
+                type: string
+                description: The image for the executionenvironment
+              organization:
+                type: string
+                description: The organization for the executionenvironment
+              pull:
+                type: string
+                description: The image pull behavior for the executionenvironment
+              request_timeout:
+                type: string
+                description: The request_timeout for the executionenvironment
+              state:
+                type: string
+                description: The state for the executionenvironment
+              validate_certs:
+                type: string
+                description: The validate_certs for the executionenvironment
```

Comment:
The CRD is missing a connection_secret variable that will allow to specify a secret with the token and host for the AAP.


```
connection_secret:
                description: |
                  A k8s secret that contains an access token for AWX. To create an access token see these docs: https://docs.ansible.com/automation-controller/4.1.0/html/userguide/applications_auth.html#add-tokens.
                type: string
```
I copied this from [another CRD](https://github.com/ansible/awx-resource-operator/blob/270c21758d82a745376f9e38ac43d92380d0f593/config/crd/bases/tower.ansible.com_ansibleinstancegroups.yaml#L66)

Reply:
Added property.